src/components/common/TaxCalculator/TaxCalculator.vue

```diff
@@ -49,11 +49,37 @@ const handleFileChange = (event: Event) => {
 const handleDeleteFile = (file: File) => {
   selectedFiles.value = selectedFiles.value.filter((selectedFile) => selectedFile !== file)
 }
+
+const isValidInput = (value: string) => {
+  const chars = '1234567890+-'
+
+  for (const char of value) {
+    if (!chars.includes(char)) {
+      return false
+    }
+  }
```

Comment:
Не не давай в функциональном стиле)) 
И вынеси из функции `const chars = '1234567890+-'` иначе в таком случае константа создается на каждый вызов функции.

```suggestion
  const checkIsValid = (value) => value.split('').every(char => chars.includes(char))
```